autonomi-cli/src/commands/wallet.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use autonomi::Multiaddr;
+use autonomi::wallet::*;
+use color_eyre::{
+    eyre::{bail, Context},
+    Result,
+};
+
+
+pub fn create(peers: Vec<Multiaddr>) -> Result<()> {
```

Comment:
I don't think we'd need `peers` argument for `create/balance`. 
`balance` uses just the evm network. the `peers` is used to bootstrap into OUR network (which we don't have to, for these commands). `create` does not need any network.

Reply:
Agreed